locales/hin.json

```diff
@@ -411,8 +418,9 @@
       "verificationFailed": {
         "title": "एक त्रुटि पाई गई!",
         "goBackButton": "वापस जाओ",
-        "technicalError": "तकनीकी त्रुटि के कारण, हम कार्ड डाउनलोड करने में असमर्थ रहे।",
-        "networkError": "अस्थिर इंटरनेट कनेक्शन के कारण, हम कार्ड डाउनलोड करने में असमर्थ थे। कृपया अपने इंटरनेट कनेक्शन की जाँच करें।"
+        "ERR_GENERIC": "तकनीकी त्रुटि के कारण, हम कार्ड डाउनलोड करने में असमर्थ रहे।",
+        "NETWORK_ERROR": "अस्थिर इंटरनेट कनेक्शन के कारण, हम कार्ड डाउनलोड करने में असमर्थ थे। कृपया अपने इंटरनेट कनेक्शन की जाँच करें।",
```

Comment:
Why this error code `NETWORK_ERROR` is different than eng language. In eng it is `ERR_NETWORK`

Reply:
Updated